### ISSUE TITLE
Mongoose multiple sort fields not working

### ISSUE THREAD

**Issue by incode4it** [doug-martin/nestjs-query#785]
**Describe the bug**

If you use mongoose and pass multiple sort fields in the `sorting` param of the query it throws the error `sort() only takes 1 Argument`

**To Reproduce**
Query example:

```
query {
  orders(
    sorting: [{field: processed, direction: ASC}, {field: createdAt, direction: DESC}]
  ) {
    name
    createdAt
    processed
  }
}
```

Response: 
```
{
  "errors": [
    {
      "message": "sort() only takes 1 Argument",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "orders"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "Error: sort() only takes 1 Argument",
            "    at model.Query.Query.sort (/home/ion/Dev/admin-panel-api/node_modules/mongoose/lib/query.js:2577:11)",
            "    at model.Query.Query.setOptions (/home/ion/Dev/admin-panel-api/node_modules/mquery/lib/mquery.js:186:20)",
            "    at model.Query.Query.setOptions (/home/ion/Dev/admin-panel-api/node_modules/mongoose/lib/query.js:1375:32)",
            "    at Function.find (/home/ion/Dev/admin-panel-api/node_modules/mongoose/lib/model.js:2040:6)",
            "    at MongooseQueryService.query (/home/ion/Dev/admin-panel-api/node_modules/@nestjs-query/query-mongoose/dist/src/services/mongoose-query.service.js:44:27)",
            "    at Service.query (/home/ion/Dev/admin-panel-api/node_modules/@nestjs-query/core/dist/src/services/assembler-query.service.js:39:68)",
            "    at /home/ion/Dev/admin-panel-api/node_modules/@nestjs-query/query-graphql/dist/src/resolvers/read.resolver.js:45:69",
            "    at Function.createFromPromise (/home/ion/Dev/admin-panel-api/node_modules/@nestjs-query/query-graphql/dist/src/types/connection/array-connection.type.js:11:24)",
            "    at AutoResolver.queryMany (/home/ion/Dev/admin-panel-api/node_modules/@nestjs-query/query-graphql/dist/src/resolvers/read.resolver.js:45:31)",
            "    at processTicksAndRejections (internal/process/task_queues.js:97:5)",
            "    at async target (/home/ion/Dev/admin-panel-api/node_modules/@nestjs/core/helpers/external-context-creator.js:76:28)",
            "    at async /home/ion/Dev/admin-panel-api/node_modules/@nestjs/core/helpers/external-proxy.js:9:24"
          ]
        }
      }
    }
  ],
  "data": null
}
```

**Expected behavior**
No Error

**Desktop**
 - Node Verson 12.16.1
 - Nestjs-query Version 0.21.2

**Comment by viennv1709** [comment]
I'm facing with this issue as well

**Comment by doug-martin** [comment]
@incode4it @viennv1709 this is fixed in `v0.25.0` I ended up fixing this while implementing `groupBy` in aggregate queries. 

Thank you for the bug report!
